lib/toml/dumper.rb

```diff
@@ -5,21 +5,32 @@ class Dumper
     def initialize(hash)
       @toml_str = ''
 
-      visit(hash, '')
+      visit(hash, [])
     end
 
     private
 
-    def visit(hash, prefix)
+    def visit(hash, prefix, extra_brackets = false)
```

Comment:
Method has too many lines. [36/25]<br>Perceived complexity for visit is too high. [12/7]